src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -200,6 +282,14 @@ function toggle(a, recursive) {
         options[i] = x[i];
     }
 
+    var delay = (function () {
+        var timer = 0;
```

Comment:
this implementation looks weird to me, because it uses a single timer for all callbacks, making it a broken debounce implementation